src/EditorFeatures/Core/FindUsages/FindUsagesHelpers.cs

```diff
@@ -72,7 +72,7 @@ public static async Task<(ISymbol symbol, Project project, ImmutableArray<ISymbo
                 symbol, project, cancellationToken).ConfigureAwait(false);
 
             var filteredSymbols = implementations.WhereAsArray(
-                s => s.Locations.Any(l => l.IsInSource));
+                s => (!s.IsAbstract || s is ITypeSymbol) && s.Locations.Any(l => l.IsInSource));
```

Comment:
i find this expression pretty confusing to make sense of.  IMO, it would benefit from just being converted to a normal block body, along with comments explaining why these checks are in place.

Reply:
i also don't think this is the right place for this change.  this is the low level code that actually finds implementations right?  in that regard, it is doing the right thing, it is providing implementations of the member.  And, in C#, an abstract method should be considered an implementation of something (for example, it can be the impl of an interface method).

If we want to change how the *feature* works, we should do that elsewhere.

Reply:
The interface case is a good example. That makes me think either the behavior before the PR was better, or we need a different algorithm altogether.

In scenarios with interfaces, I wonder if there should be a difference between invoking GoToImplementation on `I.M` (should find abstract `Base.M` and concrete `Derived.M`) versus invoking it on `Base.M` or `Derived.M` (should only find concrete `Derived.M`).